.github/workflows/build_and_test.yml

```diff
@@ -562,7 +562,9 @@ jobs:
         perl -ni.bak -e "print unless /opendds/" ACE/bin/MakeProjectCreator/config/default.features # remove opendds features from here, let individual build configure scripts handle those
         cd $GITHUB_WORKSPACE/OpenDDS
         find . -iname "*\.o" | xargs rm
-        rm build/target/setenv.sh
+        cd build/target
+        rm -rf !("ACE_TAO")
+        cd $GITHUB_WORKSPACE/OpenDDS
         git clean -xdfn | cut -d ' ' -f 3- | xargs tar cvf ../${{ github.job }}.tar
```

Comment:
I agree with what we suggested in the meeting -- we probably just need to run tar with `build/target/ACE_TAO` and `build/host/ACE_TAO` and remove the `git clean` line to create the archive.

Reply:
Since there are a lot of symlinks in the build directories, you could try using `find` to filter them out:
```
find build/host/ACE_TAO build/target/ACE_TAO -type f | xargs tar .....
```